presidio-analyzer/presidio_analyzer/predefined_recognizers/aba_routing_recognizer.py

```diff
@@ -0,0 +1,60 @@
+from presidio_analyzer import Pattern, PatternRecognizer
+
+
+class AbaRoutingRecognizer(PatternRecognizer):
+    """
+    Recognizes American Banking Association (ABA) routing number.
+    Also known as routing transit number (RTN) and used to identify financial
+    institutions and process transactions.
+    """
+
+    PATTERNS = [
+        Pattern("ABA routing number (weak)", r"\b[0123678]\d{8}\b", 0.05,),
+        Pattern("ABA routing number",
+                r"\b[0123678]\d{3}-\d{4}-\d\b",
+                0.3,),
```

Comment:
do these patterns need to be named differently?
Can I name both of these patterns `ABA routing number`?
How should we choose accuracy? I saw that sometimes its 5% and sometimes its 30%

Reply:
The name is mainly for readability. Adding the word "weak" could help future users understand that this might introduce false positives.

The best way to choose accuracy is to test this recognizer with the Presidio evaluation framework (https://github.com/microsoft/presidio-research). There it can be evaluated using real or synthetic datasets and see how well the coverage (recall) is, as well as how many false positives this recognizer introduces.

Reply:
Okay good, i'll get this other repo set up locally and test my analyzers with it.
Thanks